templates/master.yaml.tmpl

```diff
@@ -750,7 +750,7 @@ systemd:
       Environment="NAME=%p.service"
       Environment="NETWORK_CONFIG_CONTAINER="
       ExecStartPre=/usr/bin/mkdir -p /opt/bin/
-      ExecStartPre=/usr/bin/docker pull $IMAGE
+      ExecStartPre=-/usr/bin/docker pull $IMAGE
       ExecStartPre=-/usr/bin/docker stop -t 10 $NAME
       ExecStartPre=-/usr/bin/docker rm -f $NAME
```

Comment:
```suggestion
      ExecStartPre=-/usr/bin/docker stop -t 10 $NAME
      ExecStartPre=-/usr/bin/docker rm -f $NAME
      ExecStartPre=-/usr/bin/docker pull $IMAGE
```

Reply:
Not sure if the order matters, but just to be consistent.

Reply:
Fine with me :+1: